ansible_collections/arista/avd/examples/cv-pathfinder/group_vars/SITE1.yml

```diff
@@ -36,6 +36,8 @@ wan_router:
     uplink_switches: [site1-border1, site1-border2]
     uplink_type: p2p-vrfs # (1)!
     bgp_as: 65000
+    # Disabling EVPN role. This won't be necessary in AVD 6.0.
+    evpn_role: none
```

Comment:
It would be better if we could avoid this and scope the preview toggle to only concern the site 4 wan routers.